src/client/src/Stream.h

```diff
@@ -413,6 +413,8 @@ struct __KinesisVideoStream {
     // Whether to set the non-key frames as "SKIP" when we have a not-yet-completed errored fragment
     BOOL skipNonKeyFrames;
 
+    BOOL allowToCreate;
```

Comment:
Perhaps for consistency call it the same as in stream caps = allowStreamCreation?

Reply:
Agree, done.